File: backend/src/013-db/mysql/connect.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

# sudo apt-get install python-mysqldb
# sudo apt-get install python3-dev libmysqlclient-dev
# pip3 install mysqlclient
from sqlalchemy import *

USER = "root"
PASSWORD = ""
HOST = "localhost"
PORT = 3306
DATABASE_NAME = ""

URL = "mysql://%s:%s@%s:%d/%s?charset=utf8" % (USER, PASSWORD, HOST, PORT, DATABASE_NAME)
engine = create_engine(URL, pool_recycle=PORT)

with engine.connect() as db:
    print(db)

"""
Fuente:
https://es.stackoverflow.com/questions/34346/como-conectarme-con-sqlalchemy-a-una-basededatos
"""

File: backend/src/013-db/mysql/connect.js
"use strict"

var mysql = require('mysql');  // npm install mysql

var USER = "root",
    PASSWORD = "",
    HOST = "localhost",
    PORT = 3306,
    DATABASE_NAME = "";

var conn = mysql.createConnection({
  host: HOST,
  port: PORT,
  user: USER,
  password: PASSWORD,
  database: DATABASE_NAME
});

conn.connect(function(err) {
  if (err) throw err;
  console.log("Connected!");
});

/* Fuente:
 * https://www.w3schools.com/nodejs/nodejs_mysql.asp
 */
